{"uuid": "4c6a2b17-2292-4068-9b52-9bfc964fbd2b", "children": ["21d1057a-5915-449a-a8de-26a2f3d9ae1f"], "befores": [{"name": "program_management_set_up", "status": "broken", "statusDetails": {"message": "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='test.hkciot.com', port=443): Max retries exceeded with url: /cuteview/program/getInPage (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002031A7DE710>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))\n", "trace": "  File \"D:\\sw_install\\python-install\\lib\\site-packages\\pluggy\\_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\sw_install\\python-install\\lib\\site-packages\\_pytest\\fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\sw_install\\python-install\\lib\\site-packages\\_pytest\\fixtures.py\", line 909, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"D:\\ui\\test_case\\test_program_management.py\", line 24, in program_management_set_up\n    programlist.get_program_list_ids()\n  File \"D:\\ui\\common\\set_up_program_list.py\", line 24, in get_program_list_ids\n    response = requests.post(url=base_url,headers=self.headers,data=json.dumps(body))\n  File \"D:\\sw_install\\python-install\\lib\\site-packages\\requests\\api.py\", line 115, in post\n    return request(\"post\", url, data=data, json=json, **kwargs)\n  File \"D:\\sw_install\\python-install\\lib\\site-packages\\requests\\api.py\", line 59, in request\n    return session.request(method=method, url=url, **kwargs)\n  File \"D:\\sw_install\\python-install\\lib\\site-packages\\requests\\sessions.py\", line 587, in request\n    resp = self.send(prep, **send_kwargs)\n  File \"D:\\sw_install\\python-install\\lib\\site-packages\\requests\\sessions.py\", line 701, in send\n    r = adapter.send(request, **kwargs)\n  File \"D:\\sw_install\\python-install\\lib\\site-packages\\requests\\adapters.py\", line 565, in send\n    raise ConnectionError(e, request=request)\n"}, "start": 1714992970540, "stop": 1714992970540}], "start": 1714992970540, "stop": 1714992970603}